pkg/netutils/netutils.go

```diff
@@ -282,7 +282,7 @@ func isExist(mmap map[string]bool, key string) bool {
 //
 // The 0 will be returned when both minRate and defaultMinRate both are <=0.
 func CalculateTimeout(fileLength int64, minRate int, defaultMinRate int, reservedTime time.Duration) time.Duration {
-	// ensure the minRate to avoid trigger panic when minRate equals zero
+	// ensure the minRate to avoid trigger panic when minRate equals zero.
```

Comment:
I think there is no need to add a period when the comment is not a complete sentence.